src/Features/Traits/Dispatcher.php

```diff
@@ -4,6 +4,82 @@
 
 namespace Blumilk\BLT\Features\Traits;
 
+use Blumilk\BLT\Helpers\ArrayHelper;
+use Blumilk\BLT\Helpers\RecognizeClassHelper;
+use Illuminate\Contracts\Bus\Dispatcher as BusDispatcher;
+use Illuminate\Contracts\Container\BindingResolutionException;
+use Illuminate\Contracts\Events\Dispatcher as EventDispatcher;
+use Illuminate\Support\Testing\Fakes\BusFake;
+use Illuminate\Support\Testing\Fakes\EventFake;
+
 trait Dispatcher
 {
+    use Application;
+
+    /**
+     * @Given Bus is running
+     * @throws BindingResolutionException
+     */
+    public function runBus(): void
+    {
+        $busFake = new BusFake($this->getContainer()->make(BusDispatcher::class));
+        $this->getContainer()->instance(BusDispatcher::class, $busFake);
+    }
+
+    /**
+     * @Given events are faked
+     * @throws BindingResolutionException
+     */
+    public function fakeEvents(): void
+    {
+        $eventFake = new EventFake($this->getContainer()->make(EventDispatcher::class));
+        $this->getContainer()->instance(EventDispatcher::class, $eventFake);
+    }
+
+    /**
+     * @When I dispatch :count :objectName jobs with parameters: :parameters
+     * @When I dispatch :count :objectName jobs
+     * @when I dispatch :objectName job
+     * @When I dispatch :count of :objectName events with parameters: :parameters
+     * @When I dispatch :count of :objectName events
+     * @when I dispatch :objectName event
+     * @When I dispatch :count of :objectName with parameters: :parameters
+     * @When I dispatch :count of :objectName
+     * @when I dispatch :objectName
+     */
+    public function dispatchObject(string $objectName, int $count = 1, string|array $parameters = []): void
+    {
+        $parameters = ArrayHelper::toArray($parameters);
+
+        $objectClass = RecognizeClassHelper::recognizeObjectClass($objectName);
+        $objectType = RecognizeClassHelper::guessType($objectName);
+
+        if ($objectType === "Job") {
+            $objectType = "Bus";
+        }
```

Comment:
I don't get it. Why `guessType` cannot return `"Bus"`?

Reply:
I wanted helper to be more universal, wouldn't be there different usages requiring job to be recognized as one? I can change it though.

Reply:
I'm just curious, as you used the same condition two times. Maybe another class extending main guesser would be a good approach?

Reply:
Should be good now.